### PR TITLE
feat: Looker 21.10 bindings

### PR DESCRIPTION
**Pull request by joeldodge79**

### CODE REVIEW

File: release-please-config.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "plugins": ["node-workspace"],
   "bump-minor-pre-major": true,
   "bump-patch-for-minor-pre-major": true,
+  "release-as": "21.10.0",
```
Comment (joeldodge79) [inline release-please-config.json:5]: this is the top level key we set only for new Looker binding's releases (and then remove it after the release-pr is merged)

File: release-please-config.json [hunk 2]
```diff
@@ -12,11 +13,11 @@
     "packages/hackathon": { },
     "packages/run-it": { "release-as": "" },
     "packages/sdk": { },
-    "packages/sdk-codegen": { "release-as": "21.0.20" },
-    "packages/sdk-codegen-scripts": { "release-as": "21.0.21" },
-    "packages/sdk-codegen-utils": { "release-as": "21.0.13"},
+    "packages/sdk-codegen": { "release-as": "21.0.21" },
+    "packages/sdk-codegen-scripts": { "release-as": "21.0.22" },
+    "packages/sdk-codegen-utils": { "release-as": "21.0.12"},
```
Comment (joeldodge79) [inline release-please-config.json:18]: current version is `21.0.11`, not sure why i'd previously set the next version to be `21.0.13`

File: release-please-config.json [hunk 3]
```diff
@@ -12,11 +13,11 @@
...
     "packages/sdk-node": { },
-    "packages/sdk-rtl": { "release-as": "21.0.16" },
+    "packages/sdk-rtl": { "release-as": "21.0.17" },
```
Comment (joeldodge79) [inline release-please-config.json:20]: These changes are just to cover the next release version for `packages/sdk-*`. None of those packages should be affected by this PR so maybe it's distracting to have these changes in here but I thought I'd knock it out now that I had the file open

**Review by joeldodge79 — COMMENTED** [review]
@jkaster some inline comments. I glanced through the TS and Python changes and they seem fine.

File: csharp/rtl/Constants.cs [hunk 4]
```diff
@@ -61,7 +61,7 @@ public struct Constants
 
         public const string DefaultApiVersion = "4.0";
         public const string AgentPrefix = "CS-SDK";
-        public const string LookerVersion = "21.8";
+        public const string LookerVersion = "21.10";
```
Comment (jkaster) [inline csharp/rtl/Constants.cs:64]: 👍🏼

File: release-please-config.json (hunk 1 above)
Comment (jkaster) [reply release-please-config.json:5]: Thanks for the reminder

File: release-please-config.json (hunk 3 above)
Comment (jkaster) [reply release-please-config.json:20]: Works for me

**Review by jkaster — APPROVED** [review]
LGTM. Thanks for the comments.

**Pull request merged**